tests/test_integrated_channels/test_sap_success_factors/test_transmitters/test_learner_data.py

```diff
@@ -223,3 +223,11 @@ def test_transmit_by_course_id_success(self):
         assert payloads[0].error_message == 'error occurred'
         assert payloads[1].status == '200'
         assert payloads[1].error_message == ''
+
+    def test_override_channel_settings(self):
```

Comment:
Thought- is it worth it to leave a comment on the base mixin/here/somewhere about how if anyone in the future wants to override the default mixin they should add a test like this to the channel?

Reply:
yeah I think best place for that is the docstring on the mixin itself